CHANGELOG.md

```diff
@@ -22,6 +23,100 @@
  - [0.0.5](#005---20180602)
  - [0.0.4 and prior](#004-and-prior)
 
+## [0.10.0] - someday
+
+#### Breaking changes
+
+- Ingress resources now require `kubernetes.io/ingress.class` annotations by
+  default. Kong recommends adding this annotation to Ingresses that previously
+  did not have it, but you can override this change and instruct the controller
+  to process Ingresses without this annotation if desired. See the [ingress
+  class documentation](https://example.com/link-tbd) for details.
+  [#767](https://github.com/Kong/kubernetes-ingress-controller/pull/767)
+- KongConsumer resources now require `kubernetes.io/ingress.class` annotations
+  by default. This change can also be overriden using a flag.
+  [#767](https://github.com/Kong/kubernetes-ingress-controller/pull/767)
+- TCPIngress resources now require `kubernetes.io/ingress.class` annotations.
+  This change _cannot_ be overriden.
+  [#767](https://github.com/Kong/kubernetes-ingress-controller/pull/767)
+- CA certificate secrets now require `kubernetes.io/ingress.class` annotations.
+  This change _cannot_ be overriden.
+  [#815](https://github.com/Kong/kubernetes-ingress-controller/pull/815)
+- Removed support for global KongPlugin resources. You must now use
+  KongClusterPlugin resources for global plugins. You should run
+  `kubectl get kongplugin -l global=true --all-namespaces` to list existing
+  global KongPlugins to find and convert them before upgrading. The controller
+  will also log a warning if it finds any global KongPlugins that are still in
+  place.
+  [#751](https://github.com/Kong/kubernetes-ingress-controller/pull/751)
+
+
+#### Added
+
+- Added support for [Ingress
+  v1](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1453-ingress-api#summary-of-the-proposed-changes).
+  [#TBD](https://github.com/Kong/kubernetes-ingress-controller/pull/TBD).
+- Added support for the port mapping functionality in Kong versions 2.1 and
+  newer in example manifests. This feature [improves Kong's functionality when
+  behind a load balancer that uses different ports than Kong's proxy
+  listens](https://github.com/Kong/kong/pull/5861).
+  [#753](https://github.com/Kong/kubernetes-ingress-controller/pull/753)
+- Added support for the `ingress.kubernetes.io/force-ssl-redirect` annotation.
+  [#745](https://github.com/Kong/kubernetes-ingress-controller/pull/745)
+- Transitioned to structured logging.
+  [#748](https://github.com/Kong/kubernetes-ingress-controller/pull/748)
+- Added flags to enable processing of Ingress and KongConsumer resources
```

Comment:
I'll highly recommend to organize this is a little differently.
Instead, have a bullet point on "Following new flags have been introduced" and then have a sub-point to document each of the flag that has been introduced in this release.
This helps with having a single document which can tell us when was which configuration and feature introduced.